lua/keymappings.lua

```diff
@@ -1,109 +1,157 @@
-local opts = {
```

Comment:
We go back to mixing logic and data/configuration :(

Reply:
this doesn't notify anyone btw 😅 
you might want to tag someone if you want them to notice

Reply:
I thought it was just redundant for default keymaps (we want them silent and non-recursive). I can restore these opts if you think it's better though. It's not a big deal for me.

I also purposefully didn't remove `load()` and `load_mode()`, so we can always re-use them again if you'd like.